components/Repositories/RossChart.tsx

```diff
@@ -79,57 +127,32 @@ export default function RossChart({ stats, rangedTotal, isLoading, error, range,
             />
             <Tooltip content={CustomTooltip} filterNull={false} />
             <CartesianGrid vertical={false} strokeDasharray="4" stroke="#E2E8F0" />
-            <Bar dataKey="internal" stackId="a" fill="#1E3A8A" />
-            <Bar dataKey="recurring" stackId="a" fill="#2563EB" />
-            <Bar dataKey="new" stackId="a" fill="#60A5FA" />
+            {filterInternal && <Bar dataKey="internal" stackId="a" fill="#1E3A8A" />}
+            {filterRecurring && <Bar dataKey="recurring" stackId="a" fill="#2563EB" />}
+            {filterOutside && <Bar dataKey="new" stackId="a" fill="#60A5FA" />}
           </BarChart>
         )}
       </ResponsiveContainer>
+
+      <section className="flex gap-4 w-full px-4 justify-center items-center text-sm">
+        <h3>Filter:</h3>
+        <label className="flex gap-2">
+          <input type="checkbox" checked={filterOutside} onChange={() => setFilterOutside(!filterOutside)} />
+          Outside
+        </label>
+        <label className="flex gap-2">
+          <input type="checkbox" checked={filterRecurring} onChange={() => setFilterRecurring(!filterRecurring)} />
+          Recurring
+        </label>
+        <label className="flex gap-2">
+          <input type="checkbox" checked={filterInternal} onChange={() => setFilterInternal(!filterInternal)} />
+          Internal
+        </label>
+      </section>
```

Comment:
Refactor (blocking): Use a [fieldset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) and OpenSauced's design system checbox.

```suggestion
      <fieldset className="grid grid-cols-[auto_1fr] gap-2 items-center text-sm px-2 py-1">
        <legend className="contents px-1">Filter by:</legend>
        <span className="flex gap-2 items-center">
          <Checkbox label="Outside" checked={filterOutside} onChange={() => setFilterOutside(!filterOutside)} />
          <Checkbox label="Recurring" checked={filterRecurring} onChange={() => setFilterRecurring(!filterRecurring)} />
          <Checkbox label="Internal" checked={filterInternal} onChange={() => setFilterInternal(!filterInternal)} />
        </span>
      </fieldset>
```

Reply:
Using `<h3>` instead of `<legend>` since its styling is based on the fieldset; can't make it inline like the original design.